thirdparty/faiss/faiss/IndexIVFPQFastScan.cpp

```diff
@@ -1332,7 +1332,8 @@ void IndexIVFPQFastScan::range_search_implem_12(
     TIC;
 
     // prepare the result handlers
-    std::unique_ptr<RangeSearchResultHandler<C, true>> handler(new RangeSearchResultHandler<C, true>(result, radius, 0, bitset));
+    float radius_thresh = C::is_max ? radius * 1.2 : radius * 0.6;  // to make range search result more accurate
```

Comment:
Such a change should never be done on the Faiss side, but on the Knowhere side. Please move it to `ivf.cc`.